### PR TITLE
random name generation

### PR DESCRIPTION
**Pull request by YazeedLoonat**

### CODE REVIEW

File: src/dictionary.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const https = require("https");
+const fs = require("fs");
+const generateList = async () => new Promise((resolve, reject) => {
+    let wordList = [];
+    if (fs.existsSync("../dictionary.json")) {
+        wordList = fs.readFileSync("../dictionary.json");
```
Comment (prosif) [inline src/dictionary.js]: I'd like to have a local cache dir, maybe homegames/local by default that we store files to, but this is okay for now

File: src/dictionary.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        resolve(JSON.parse(wordList));
+    } else {
+        https.get("https://raw.githubusercontent.com/dwyl/english-words/master/words_alpha.txt", (res) => {
```
Comment (prosif) [inline src/dictionary.js]: Another task we can make is to have a separate config file where this dictionary URL is set

File: src/dictionary.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            res.on("data", (d) => {
+                wordList = [ ...wordList, ...d.toString().split(/\r?\n/)];
+            });
+            res.on("end", () => {
+                fs.writeFile("../dictionary.json", JSON.stringify(wordList), (err) => {
```
Comment (prosif) [inline src/dictionary.js]: If we use a config file, we can avoid hardcoding "dictionary.json"

File: src/dictionary.js (hunk 1 above)
Comment (YazeedLoonat) [reply src/dictionary.js]: I agree and moved stuff to a local directory

File: src/dictionary.js (hunk 3 above)
Comment (YazeedLoonat) [reply src/dictionary.js]: I agree and spun up issue: https://github.com/prosif/homegames/issues/17
to track that

File: src/dictionary.js (hunk 2 above)
Comment (YazeedLoonat) [reply src/dictionary.js]: I agree and spun up issue: https://github.com/prosif/homegames/issues/17
to track that

**Pull request merged**